frontend/mr-admin-flate/src/components/filter/Tiltakstypefilter.tsx

```diff
@@ -0,0 +1,90 @@
+import { Search, Select } from "@navikt/ds-react";
+import classNames from "classnames";
+import { useAtom } from "jotai";
+import { SorteringTiltakstyper } from "mulighetsrommet-api-client";
+import { ChangeEvent } from "react";
+import { tiltakstypefilter, paginationAtom } from "../../api/atoms";
+import styles from "./Filter.module.scss";
+
+export function Tiltakstypefilter() {
+  const [sokefilter, setSokefilter] = useAtom(tiltakstypefilter);
+  const [, setPage] = useAtom(paginationAtom);
+
+  const resetPagination = () => {
+    setPage(1);
+  };
+
+  return (
+    <div className={classNames(styles.filter_container)}>
```

Comment:
Trenger vi classNames her når et bare er ett classname?

Reply:
Nei, godt spottet. Det var opprinnelig to klasser. Jeg fjerner!